tests/libtest.sh

```diff
@@ -73,7 +73,7 @@ cleanup() {
     outfile=$4
   fi
 
-
+  rm -rf /var/lib/container-storage-setup
```

Comment:
run-test.sh checks it. So we just need to add some logic there. look at check_metadata()